extension/src/CSV/ImportXliffTSV.ts

```diff
@@ -0,0 +1,31 @@
+import { Xliff } from "../XLIFFDocument";
+import { CSV } from "./CSV";
+
+const requiredHeaders: string[] = ["Id", "Source", "Target"];
+
+export function importXliffTSV(updateXlf: Xliff, csvPath: string): number {
+    let csv = new CSV();
+    let updatedTargets: number = 0;
+    csv.importFileSync(csvPath);
+    testRequiredHeaders(csv.headers);
+    csv.lines.forEach(line => {
+        let values = { id: line[0], source: line[1], target: line[2] }
+        let transunit = updateXlf.getTransUnitById(values.id);
+        if (transunit.source !== values.source) {
+            throw new Error(`Sources doesn't match for id ${transunit.id}`);
```

Comment:
might be an idea to write both sources as well?

Reply:
Sure, it might not be super pretty in all cases but it's helpful information :)